signer/signer.go

```diff
@@ -215,10 +215,13 @@ type NamedSignedFile namedFile
 
 // isValidUnsignedFilename
 func isValidUnsignedFilename(filename string) error {
-	if !regexp.MustCompile(`^[a-zA-z0-9]`).MatchString(filename) {
+	if !regexp.MustCompile(`^[a-zA-Z0-9]`).MatchString(filename) {
 		return fmt.Errorf("unsigned filename must start with an alphanumeric character")
 	}
-	if !regexp.MustCompile(`^[-_\.a-zA-Z0-9]{1,256}$`).MatchString(filename) {
+	if !regexp.MustCompile(`[a-zA-Z0-9]$`).MatchString(filename) {
```

Comment:
@ahal - this will affect all gpg2 zip file contents signing. RelEng is, I believe, the only user of that signing mode. Can you sign off that the change won't adversely impact any other signing, please?

Reply:
Assuming you mean this only impacts signing formats using the `files` endpoint, this should be safe. Afaict, VPN is the only consumer using that endpoint with gpg signing. Further, that format only signs files with specific file extensions (which all end in alphanumeric characters).

Even if this does impact other things other than `files`, the chance of it causing problems seems exceedingly small. I can't think of any files we'd want to sign that would end in non alphanumeric characters.